### DISCUSSION TITLE
How do I provide alternate forms of identification for the student discount?

### DISCUSSION THREAD

**Discussion by watanabekyonosuke** [community/community#120365] [GitHub Education]
## inquiry

I am trying to get access to github's university discount, but my student ID doesn't have a date on it, so my initial application with the ID was rejected. The application also doesn't take other docs. How should I provide other docs - transcript, receipts, etc?

**Comment by samus-aran** [comment]
Hi @watanabekyonosuke 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I have a double-sided Student ID with the date on the other side. How do I submit this as evidence?](https://github.com/orgs/community/discussions/111352#double-sided-student-id-with-the-date-on-the-other-side)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Reply by watanabekyonosuke** [reply]
Thanks for the help, I submitted an online enrollment history, but that was about a week ago. When should I expect a response? How can I check the status of my application.

**Reply by samus-aran** [reply]
👍 

Hi @watanabekyonosuke 

You should find your answers on our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. I've included the section that I feel would best assist.

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

You’ll get an email notification once your academic benefits are ready, or you can check your status [here ](https://education.github.com/discount_requests/pack_application).

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
